oam/install_scripts/mcs-loadbrm.service.in

```diff
@@ -1,11 +1,11 @@
 [Unit]
 Description=loadbrm
-PartOf=mcs-workernode.service
+PartOf=mcs-workernode@.service
```

Comment:
I wonder how does this work in case of multiple services on the same node? Would mcs-loadbrm restarts if any of the service instances restarts?

Reply:
I have addressed this with a change explicitly stating instances of workernode